packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/translations/Messages_fr_FR.json

```
@@ -1,6 +1,21 @@
 {
-  "ip_agora_table_header_offer": "Offre",
+  "ip_agora_table_header_offer": "Choisisser votre offre",
```

`Choisisser` form doesn't exist
```suggestion
  "ip_agora_table_header_offer": "Choisissez votre offre",
```
or 
```suggestion
  "ip_agora_table_header_offer": "Choisir votre offre",
```